### DISCUSSION TITLE
Submenu icons

### DISCUSSION THREAD

**Discussion by kenlas** [livewire/flux#897] [Feature Requests] [answered]
Hey there guys! Started using Flux and really like it. I was wondering if dropdown submenus should also have an icon property? Now when the sibling menu items have an icon, the submenu does not have an option to set the icon, too.

![image](https://github.com/user-attachments/assets/576df08f-8ff0-450a-8f4a-27c51082f0d2)

**Comment by joshhanley** [comment, marked as answer]
@kenlas thanks for the request! It's the same as issue #889 and a PR to add this was merged today and will be in the next release 🙂
